src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -872,8 +877,9 @@ private static FormulaValue PositiveNumericNumberChecker(IRContext irContext, in
             {
                 var number = numberArg.Value;
                 if (number < 0)
-                {
-                    return CommonErrors.ArgumentOutOfRange(irContext);
+                {
+                    // The caller will localize this error message.
```

Comment:
>/ The caller will localize this error message [](http://example.com/codeflow?start=21&length=45)

What do you mean that this will be localized by the caller?

Reply:
This has been removed due to shifting the approach.